### PR TITLE
Rebranding observing-squad

### PR DESCRIPTION
**Pull request by miiu96**
- Updated docker images
- Updated GitHub action and changed images name

### CODE REVIEW

File: utils/chain-keygenerator [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+RUN git clone https://github.com/multiversx/mx-chain-go.git
+
+#Build keygenerator
+WORKDIR /go/mx-chain-go/
+
+WORKDIR /go/mx-chain-go/cmd/keygenerator
+RUN go build
+
+# ===== SECOND STAGE ======
+FROM ubuntu:18.04
```
Comment (bogdan-rosianu) [inline utils/chain-keygenerator:12]: ubuntu 20.04 at least?

Comment (miiu96) [reply utils/chain-keygenerator:12]: changed

File: mainnet/chain-proxy [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM multiversx/chain-proxy:v1.1.27-rc1
```
[the remark this answers is not in the document]
Comment (miiu96) [reply mainnet/chain-proxy:1]: changed.

File: .github/workflows/deploy-docker.yaml [hunk 3]
```diff
@@ -30,10 +30,10 @@ jobs:
           done < ${ENV_FILE}
       - name: Build node image
         run: |
-          cd ${GITHUB_WORKSPACE}/mainnet/ && docker image build . -t "${REGISTRY_HOSTNAME}/${NODE_TAG}" -f ./elrond-node-obs
+          cd ${GITHUB_WORKSPACE}/mainnet/ && docker image build . -t "${REGISTRY_HOSTNAME}/${NODE_TAG}" -f ./chain-obs
```
[the remark this answers is not in the document]
Comment (miiu96) [reply .github/workflows/deploy-docker.yaml:33]: changed

File: mainnet/.env [hunk 4]
```diff
@@ -8,15 +8,10 @@ DISPLAY_NAME_1="MyObservingSquad-1"
...
-IP_1=10.0.0.5
-IP_2=10.0.0.4
-IP_M=10.0.0.3
+SQUAD_DIRECTORY=~/MyObservingSquad
 
-NODE_TAG=elrond-node-obs:v1.3.50.1
-PROXY_TAG=elrond-proxy:v1.1.27
+ORGANIZATION=multiversx
+
+NODE_TAG=chain-obs:v1.3.50.3
```
[the remark this answers is not in the document]
Comment (miiu96) [reply mainnet/.env:16]: changed

File: README.md [hunk 5]
```diff
@@ -151,8 +151,8 @@ export IP=10.0.0.2
 
 1. Pull the new images:
 ```
-docker pull elrondnetwork/elrond-node-obs:v1.3.50.1
-docker pull elrondnetwork/elrond-proxy:v1.1.27
+docker pull multiversx/multiversx-node-obs:v1.3.50.3
+docker pull multiversx/multiversx-proxy:v1.1.27
```
Comment (bogdan-rosianu) [inline README.md:155]: -rc1 ?

File: mainnet/docker-compose.yml [hunk 6]
```diff
@@ -2,80 +2,80 @@ version: '3'
...
    ports:
      - "8081:8080"
      - "10001:37373"
    environment:
      - SHARD=1 
    volumes:
-     - ~/MyObservingSquad/node-1/db:/go/elrond-go/cmd/node/db
-     - ~/MyObservingSquad/node-1/logs:/go/elrond-go/cmd/node/logs
-     - ~/MyObservingSquad/node-1/config:/config
+     - ${SQUAD_DIRECTORY}/node-1/db:/go/mx-chain-go/cmd/node/db
```
Comment (bogdan-rosianu) [inline mainnet/docker-compose.yml:28]: might document in the Readme or Docs how to rename volumes for a smooth transition to the new observing squad version(s)

File: README.md (hunk 5 above)
Comment (miiu96) [reply README.md:155]: fixed

File: mainnet/docker-compose.yml (hunk 6 above)
Comment (miiu96) [reply mainnet/docker-compose.yml:28]: I will do some tests to verify if the upgrade with the new squad works correctly with the previous version.

**Review by andreibancioiu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**